views/includes/page-templates/relevant-period.html

```diff
@@ -1,6 +1,7 @@
 {% set start_date = "28 February 2022" %}
+{% import "includes/date-macros.html" as dateMacros %}
```

Comment:
please delete this because is already imported in the banner

Reply:
It isn't [here](https://github.com/companieshouse/overseas-entities-web/blob/main/views/includes/page-templates/important-banner.html) shall I add the import in the banner? my only issue with that is what if we need to use this `relevant-period.html` on it's own, which I know we need for the interrupt screen at least